### PR TITLE
Matrix VV Preview (TG Port)

### PR DESCRIPTION
**Pull request by Jumps0**
### Intent of your Pull Request

Adds a preview for the Matrix VV thing

All credit to the original coder **AnturK**.
This is the second part of this PR, the orbital fix was already merged and was just waiting for the Smart Asset Cache to be merged. (**Smart Asset Cache PR is required for this to work**)

vv

#### Changelog

:cl:  AnturK
tweak: adds matrix vv preview
/:cl:

### CODE REVIEW

**Review by monster860 — DISMISSED** [review]
Theres already a matrix display right below it though, and you didnt remove it

**Comment by Jumps0** [comment]
> Theres already a matrix display right below it though, and you didnt remove it

So are you saying this PR is irreverent?

**Comment by monster860** [comment]
no I'm saying theres redundant code you need to remove

**Comment by Jumps0** [comment]
im dumb

File: code/modules/asset_cache/asset_list_items.dm [hunk 1]
```diff
@@ -356,3 +356,9 @@
 	assets = list(
 		"map.png" = fcopy_rsc(file("icons/mapimages/[SSmapping.config.map_name].png")) //AFAIK this doesn't race with SSmapping loading up.
 	)
+
+
```
Comment (JamieD1) [inline code/modules/asset_cache/asset_list_items.dm]:
```suggestion
```

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**